text/0297-deprecate-ember-logger.md

```diff
@@ -70,9 +61,15 @@ Note: None of the uses of `Ember.Logger` in `ember.js` or `ember-data` involve
 `Ember.debug`, so that issue doesn't affect the Ember.js code directly.
 
 Add deprecation warnings to the implementation: `ember-console\lib\index.js`.
-Bear in mind that the deprecation mechanism currently calls `Logger.warn`, so 
-that code should be changed _first_ or this change will be very difficult 
-to debug.
+Bear in mind that the deprecation mechanism in `ember-debug` currently calls 
+`Logger.warn`, so the `ember-debug` code should be changed _first_ or adding 
+the deprecation warning will create a deep recursion.
+
+The log and debug messages emitted by Ember for deprecations and assertions 
```

Comment:
You might want to slightly rephrase this section to specifically indicate that it is `Ember.warn` and friends that are noops in production.